test/expected/edition.out

```diff
@@ -79,36 +73,52 @@ SELECT * FROM  _timescaledb_internal.tsl_loaded();
  t
 (1 row)
 
-SELECT * FROM  _timescaledb_internal.license_expiration_time();
- license_expiration_time 
--------------------------
- infinity
+SET timescaledb.license='tsl';
+ERROR:  permission denied to set parameter "timescaledb.license"
```

Comment:
Why is this an error? I don't see a switch from superuser to unprivileged user, so why does it generate an error?